server/controllers/profile.js

```diff
@@ -0,0 +1,62 @@
+const Profile= require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+
+
+// @route PUT /profile/all
+// @desc find all profiles
+// @access Private
+
+exports.findAllProfiles= asyncHandler(async (req, res, next) => {
+  let profiles=await Profile.find()
+  res.status(200).json({
+    success:{profiles}
+  })
+})
+
+// @route POST /profile/create
+// @desc create a profile
+// @access Private
+
+exports.createProfile = asyncHandler(async (req, res, next) => {
+  
+  await Profile.create({
+    firstName:req.body.firstName,
+    lastName:req.body.lastName,
+    description:req.body.description,
+    gender:req.body.gender,
+    birthDate:req.body.birthDate,
+    phoneNumber:req.body.phoneNumber,
+    address:req.body.address,
+    isAvailable:req.body.isAvailable,
+    availability:req.body.description,
+  })
```

Comment:
We are repeating `req.body` quite a bit here. Use ES6 de-structuring to clean this up a bit.